pkg/reporter/gitlab.go

```diff
@@ -364,3 +367,26 @@ func (r *gitLabReporter) getVulnerabilitySeverity(risk insightapi.PackageVulnera
 		return SeverityUnknown
 	}
 }
+
+// getVulnerabilityDescription returns the description for a vulnerability
+// Markdown formatted
+func (r *gitLabReporter) getGitLabVulnerabilityDescription(pkg *models.Package, summary string) string {
+	pkgName := pkg.GetName()
+	pkgVersion := pkg.GetVersion()
+
+	description := fmt.Sprintf("Package **`%s@%s`**\n\n%s",
+		pkgName,
+		pkgVersion,
+		summary)
+
+	return description
+}
+
+// getVulnerabilitySolution returns the solution for a vulnerability
+// Markdown formatted
+func (r *gitLabReporter) getGitLabVulnerabilitySolution(pkg *models.Package) string {
+	latestVersion := utils.SafelyGetValue(pkg.Insights.PackageCurrentVersion)
```

Comment:
There are cases where `PackageCurrentVersion` may be empty. I think we should check that and add an appropriate message saying `No solution available for this issue` to avoid incorrect advice.